### DISCUSSION TITLE
Meshcentral public server for accessibility-test?

### DISCUSSION THREAD

**Discussion by Michael1972** [Ylianst/MeshCentral#6441] [Q&A]
Hi to the cummunity, I'm working for a blind- and visually-impaired organisation in Germany. We would like to provide self-hosted remote. Meshcentral would be a good choice for us, but before setting up our own server, I would like to know, if the interface and of course the remote-session are accessible with screenreaders. Unfortunately, the former public server at meshcentral.com/login is closed. So, is there any way to test meshcentral at a different location publicly? We are all working voluntarily, and having an impression of meshcentral concerning accessibility before setting up an own server possibly in vain would be very good. A short test-period of one or two days would be sufficient.

Thanks in advance for any help in this and best.

Michael

**Comment by si458** [comment]
from my understanding, the is alittle bit of accessibility in the web ui
as i discovered the 'accessibility' tab in firefox developer tools by mistake and you can see from this screenshot info in there
![Screenshot 2024-10-09 at 13 00 50](https://github.com/user-attachments/assets/519f2934-e490-4416-8f49-0afafb18be71)
but i think you are really gunna have to setup a meshcentral instance, add machines and then test with different browsers, accessibility devices and then let us know what does and doesnt work!
we are always happy to improve the web ui!
as it happens right now we have a team working on re-building the web ui into bootstrap which might be more accessibility friendly!
